templates/taxi/car_confirm_delete.html

```diff
@@ -6,5 +6,6 @@ <h1>Delete car?</h1>
     {% csrf_token %}
 
     <input type="submit" value="Yes" class="btn btn-danger">
+
   </form>
 {% endblock %}
```

Comment:
It's better to always end text files with a newline character. You can read why this is the case [here](https://stackoverflow.com/a/729795).